### ISSUE TITLE
[🐛 Bug]: accept_insecure_certs in Options() fails to bypass self-signed cert warnings

### ISSUE THREAD

**Issue by Lvl4Sword** [SeleniumHQ/selenium#11169]
### What happened?

Using Selenium 4.5.0, and Geckodriver 0.33.0, setting the following:
```py

options = Options()
options.accept_insecure_certs = True

```
**should** bypass self-signed cert warnings, but does not.

### How can we reproduce the issue?

```shell
options = Options()
options.headless = False
options.accept_insecure_certs = True
service = Service('/home/user/geckodriver')
ff = webdriver.Firefox(service=service, options=options)
ff.get('https://self-signed.badssl.com/')

won't bypass the insecure cert warning
```

### Relevant log output

```shell
No log output
```

### Operating System

Ubuntu 22.04 LTS

### Selenium version

Python 4.5.0

### What are the browser(s) and version(s) where you see this issue?

Firefox 106.0.1 (64-bit)

### What are the browser driver(s) and version(s) where you see this issue?

geckodriver 0.33.0

**Comment by titusfortner** [comment]
haven't had a chance to test in Python, but verified behavior works as expected in Ruby, so it isn't a geckodriver issue

**Comment by codejunction** [comment]
you can use "--allow-running-insecure-content" in options
```python
options.add_argument("--allow-running-insecure-content")
```

**Comment by Lvl4Sword** [comment]
@codejunction
That doesn't seem to work and I still get a self-signed cert warning:
> The certificate is not trusted because it is self-signed.
> Error code: MOZILLA_PKIX_ERROR_SELF_SIGNED_CERT

**Comment by Lvl4Sword** [comment]
Okay, so this is interesting.

I thought it had to do with a redirect, since what my setup looks like is I'm trying to login to a self-signed site and set it up.
Oddly enough, the first page doesn't give an error, but when signing-in it does.
So, I:
- Loaded the page and **DIDN'T** send credentials.
- Loaded https://self-signed.badssl.com/

To see if this was the case. It was not. Confused, I continued on.
Turns out that just setting up a time.sleep(5) before inputting username/password and submitting it was enough to not get a self-signed error.

Not using anything like `options.add_argument("--allow-running-insecure-content")`, `options.accept_insecure_certs = True`, or whatever else related to certificates/content either.

So, it's definitely still a bug.. just not the one I thought it was.
